Automattic/wp-calypso
# Add re-usable <ZendeskChat/> widget (component)

#### Proposed Changes

This PR simply extracts the `<ZendeskChat />` re-usable component into it's own PR, for 1, to simplify PR logic, and for 2, We (Jetpack) needed access to this component immediately so that we can deploy our pre-sales chat widgets so they are ready to go by tomorrow morning (Wed, Nov 30th)
Also extracted/copied over the unit-test associated with the ZendeskChat component.

Asana task: 1202858161751496-as-1203406293146511/f

#### Testing Instructions

I think it should be enough to inspect the code, and run the unit-test at, `yarn test-client `client/components/presales-zendesk-chat-widget/test/index.js`

#### Pre-merge Checklist





Related to #
